content/docs/concepts/architecture/index.md

```diff
@@ -64,8 +68,14 @@ This category of services reacts on Keptn events and perform: (1) initial tasks
 
 - **remediation-service:** It is used to orchestrate remediation workflows. 
 
-## Keptn services
+Execution plane services subscribe to events via a distributor sidecar, that forwards incoming `.triggered` events to them. These distributor sidecars can also be used to send `.started` and `.finished` events back to the Keptn control plane.
 
-To enrich the continuous delivery use case or to build automated operations, Keptn relies on services from the community. Those services can be easily plugged into a task sequence to extend the delivery pipeline or to further automate operations.
+As illustrated in the architecture diagram above, execution plane services can both be operated within the same cluster as Keptn, or outside of the cluster. 
+
+When they are operated within the same cluster, the services can directly access the HTTP APIs provided by the control plane services,
+without having to authenticate. In this case, the distributor sidecars directly connect themselves to the NATS cluster to subscribe to topics and send back events.
+
+When an execution plane is operated outside of the Cluster, it has the possibility to communicate with the HTTP API exposed by the `api-gateway-nginx`. In this case, every request to the API has to be authenticated using the `keptn-api` token. 
+Also, the distributor sidecars do not have the possibility to directly connect to the NATS cluster, but they can be configured to fetch open `.triggered` events from the HTTP API.
```

Comment:
Can you make a listing in the sense of: 

* When they are ...

* When an execution plane ...

Reply:
done